modules/ROOT/pages/notifications-configuration-reference.adoc

```diff
@@ -0,0 +1,116 @@
+= Notifications Configuration Reference
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: anypoint studio, esb, notifications, error messages, responses, exceptions, monitor
+
+This topic provides details on the elements to configure for notifications.
+//DOCS-4183 For more information on notifications, see xref:mule-server-notifications.adoc[Mule Server Notifications].
+
+== Notifications
+
+Registers listeners for notifications, and associates interfaces with specific events.
+
+=== Attributes of Notifications
+
+[%header%autowidth.spread]
+|===
+|Name |Description
+|`dynamic` |If the notification manager is dynamic, you can set up the listeners to be registered dynamically at runtime. Otherwise, some parts of Mule will cache the notification configurations for efficiency reasons and will not generate events for newly-enabled notifications or listeners.
+//See https://docs.mulesoft.com/mule-runtime/3.9/mule-server-notifications#registering-listeners-programmatically for more information.//
```

Comment:
Is this linking to 3.9 docs?

Reply:
It is commented out. I will remove the comments anyway.